drivers/enc28j60/enc28j60.c

```diff
@@ -470,6 +482,7 @@ static void nd_isr(netdev_t *netdev)
                 DEBUG("[enc28j60] isr: packet received\n");
                 netdev->event_callback(netdev, NETDEV_EVENT_RX_COMPLETE);
             } while (cmd_rcr(dev, REG_B1_EPKTCNT, 1) > 0);
+            cmd_bfc(dev, REG_EIR, -1, EIR_PKTIF);
```

Comment:
> After decrementing, if EPKTCNT is ‘0’, the EIR.PKTIF flag will automatically be cleared.

Citing section 7.2.4 on page 45 in the [datasheet](http://ww1.microchip.com/downloads/en/DeviceDoc/39662b.pdf). I assume this line could be dropped.

Reply:
Probabely yes, but it shouldn't cause errors if it is there. Since I don't have time to test it, I would leave it as it is for the moment. If you could test, we could decide to remove it.

Reply:
@maribu Have you tried it?

Reply:
Did so now. Works fine without manually clearing EIR_PKTIF.